test/lint-guard.spec.ts

```
@@ -0,0 +1,48 @@
+import { spawnSync } from "node:child_process";
+import { describe, expect, it } from "vitest";
+
+function runGuard(filePath: string): string {
+	const input = JSON.stringify({ tool_input: { file_path: filePath } });
+	const result = spawnSync("node", ["hooks/lint-guard.ts"], {
+		encoding: "utf-8",
+		input,
+	});
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)